src/transform/NodeUtilities.js

```diff
@@ -1,11 +1,17 @@
+// Node is undefined in Node.js
+const ElementNode = 1
```

Comment:
It may be cleaner to export a single `NODE_TYPE` constant:
<pre>
const NODE_TYPE = {
  ELEMENT_NODE: 1,
  TEXT_NODE: 3
}
</pre>

Then we can access any of the types through it: `NODE_TYPE.ELEMENT_NODE` or `NODE_TYPE.TEXT_NODE`.

Reply:
Sounds good to me. Pushed that change.